.golangci.yml

```diff
@@ -93,6 +93,13 @@ issues:
       linters:
         - stylecheck
         - noctx
+    - path: test/helper/e2e
+      linters:
+        - stylecheck
+      test: "ST1001:"
+    - path: test/helper/e2e/appclient
+      linters:
+        - noctx
```

Comment:
@s-urbaniak 
this is the smallest change set I found to get this working while still passing as many linters and rules in the moved code

Reply:
Actually I have a typo there, it is `text`, not `test`

fixing...

Reply:
done